src/makeCalculator.js

```diff
@@ -37,7 +37,26 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    result: 0,
+
+    add: (a, b) => a + b,
+    subtract: (a, b) => a - b,
+    divide: (a, b) => a / b,
+    multiply: (a, b) => a * b,
+    reset() {
+      this.result = 0;
+
+      return this;
+    },
+    operate(operation, number) {
+      this.result = operation(this.result, number);
```

Comment:
Use `operation.call(context, argument)` to call function with new context and argument.